rskj-core/src/main/java/co/rsk/trie/TrieImpl.java

```diff
@@ -589,10 +588,10 @@ private Trie retrieveNode(int n) {
         node = this.store.retrieve(localHash.getBytes());
 
         if (node == null) {
-            String strHash = localHash.toHexString();
-            logger.error(ERROR_NON_EXISTENT_TRIE_LOGGER, strHash);
-            panicProcessor.panic(PANIC_TOPIC, ERROR_NON_EXISTENT_TRIE + " " + strHash);
-            throw new TrieSerializationException(ERROR_NON_EXISTENT_TRIE + " " + strHash, null);
+            String log = String.format(ERROR_NON_EXISTENT_TRIE, localHash.toHexString());
```

Comment:
Maybe we can add some more information to the Exception message like the requested index